zanata-war/src/main/java/org/zanata/rest/service/SourceDocResourceService.java

```diff
@@ -421,7 +421,7 @@ private HLocale validateTargetLocale(LocaleId locale, String projectSlug,
         } catch (ZanataServiceException e) {
             log.warn(
                     "Exception validating target locale {} in proj {} iter {}",
-                    e, locale, projectSlug, iterationSlug);
+                    locale, projectSlug, iterationSlug);
```

Comment:
It does work. Although IntelliJ still gives warning on using it this way. 

----- Original Message -----

> From: "Sean Flanigan" <notifications@github.com>
> To: "zanata/zanata-server" <zanata-server@noreply.github.com>
> Cc: "Patrick Huang" <pahuang@redhat.com>
> Sent: Friday, March 20, 2015 2:18:02 PM
> Subject: Re: [zanata-server] rhbz1124630 - convert server REST tests after
> client jersey change (#642)

> In
> zanata-war/src/main/java/org/zanata/rest/service/SourceDocResourceService.java
> :
> > @@ -421,7 +421,7 @@ private HLocale validateTargetLocale(LocaleId locale,
> > String projectSlug,
> >          } catch (ZanataServiceException e) {
> >              log.warn(
> >                      "Exception validating target locale {} in proj {} iter
> >                      {}",
> > -                    e, locale, projectSlug, iterationSlug);
> > +                    locale, projectSlug, iterationSlug);

> Try it. It works. http://slf4j.org/faq.html#paramException

> —
> Reply to this email directly or view it on GitHub .